.ci/packer_cache.sh

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/env bash
+
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+
+# This script used to "bake" Docker images into base image for Jenkins nodes.
+
+set -eou pipefail
+
+DOCKER_CI_IMAGE=$(cd ../build/ci/ && make show-image)
+
+declare -a docker_images=("$DOCKER_CI_IMAGE")
+
+# Pull all the required docker images
+for image in "${docker_images[@]}"
```

Comment:
not sure we need to iterate on an array of 1 element 🤷‍♂

Reply:
what if you cannot pull the image because it's not built yet? will it make the CI process fail?

Reply:
If you cannot pull the docker image, then the Packer image build for ECK workers will fail but we can still relaunch the build or wait next build the next day. Meanwhile ECK workers can still use the Packer image from the day before.

Reply:
> not sure we need to iterate on an array of 1 element 🤷‍♂

In the future we will add more images, like for `kind`

Reply:
> will it make the CI process fail?

It will not affect our builds on CI